### ISSUE TITLE
Doesn't release gpu memory when paused

### ISSUE THREAD

**Issue by acoreyj** [trexminer/T-Rex#314]
Is this something t-rex can do or is it up to my system to figure out to take the memory back while t-rex is paused?

**Comment by jhorath** [comment]
He is probably going to say that it's nvidia drivers fault. there is a known memory leak/growth issue with "latest" drivers per t-rex dev. sadly he is "not keen to spend time implementing workarounds for NVML bugs" and the only recourse for his users who are basically paying for the miners via dev fee is to turn off ALL features pertaining to stats, which defeats the purpose of using any of the api.  Use this setting and it will greatly reduce if not stop the memory issues. It does for sure for my issue and may help your.   --no-nvml

**Comment by jhorath** [comment]
https://github.com/trexminer/T-Rex/issues/262

**Comment by acoreyj** [comment]
> He is probably going to say that it's nvidia drivers fault. there is a known memory leak/growth issue with "latest" drivers per t-rex dev. sadly he is "not keen to spend time implementing workarounds for NVML bugs" and the only recourse for his users who are basically paying for the miners via dev fee is to turn off ALL features pertaining to stats, which defeats the purpose of using any of the api. Use this setting and it will greatly reduce if not stop the memory issues. It does for sure for my issue and may help your. --no-nvml

I don't think this is it, stopping t-rex cleans up resources and ends vram usage but pausing it does not.

**Comment by tchesket** [comment]
I can confirm that making --no-nvml "true" in the config file does not seem to allow the miner to release the VRAM when paused. Is there any more info on this issue?
